README.md

```diff
@@ -35,7 +36,23 @@ Convert the Ruby and ActiveRecord code below into Javascript and Mongoose code:
 ```
 
 ```js
-// Your answer...
+
```

Comment:
`show: function(req){`

Using this `:` outside of an object literal will throw a syntax error.